code/__DEFINES/mode.dm

```diff
@@ -69,6 +70,12 @@
 #define MODE_CIVIL_WAR_REBEL_MAJOR "Rebel Major Victory"
 #define MODE_CIVIL_WAR_DRAW "Civil War Draw"
 
+#define MODE_SOM_MARINE_MAJOR "Marine Major Victory"
+#define MODE_SOM_MARINE_MINOR "Marine Minor Victory"
+#define MODE_SOM_SOM_MAJOR "Sons of Mars Major Victory"
+#define MODE_SOM_SOM_MINOR "Sons of Mars Minor Victory"
+#define MODE_COMBAT_PATROL_DRAW "Draw: Both sides wiped out!"
```

Comment:
```suggestion
#define MODE_COMBAT_PATROL_DRAW "DRAW: Both sides wiped out!"
```
consistancy.

Reply:
Just gonna change it to the other draw message, since draw can occur without both sides completely wiping.